provider/win_dump_utils.py

```diff
@@ -79,13 +80,12 @@ def install_windbg(test, params, session, timeout=600):
     windbg_install_cmd = utils_misc.set_winutils_letter(session,
                                                         windbg_install_cmd
                                                         % params["feature"])
-
-    session.cmd(windbg_install_cmd)
-    if not utils_misc.wait_for(lambda: check_windbg_installed(params, session),
-                               timeout=timeout, step=5):
-        test.fail("windbg tool has not been installed")
-    else:
-        LOG_JOB.info("windbg tool installation completed")
+    while check_windbg_installed(params, session) is not True:
+        status, output = session.cmd_status_output(windbg_install_cmd)
+        if status == 1:
+            time.sleep(30)
+        else:
+            LOG_JOB.info("windbg tool installation completed")
```

Comment:
I think the original utils_misc.wait_for() function is better， extending the timeout value could cover your requirement.

Reply:
Agree with you. We can only modify the WMIC queries part and keep the current timeout value.